src/views/UniverseView/Components/Filter/Filter.jsx

```diff
@@ -16,21 +16,21 @@ export function Filter ({ setFilters, fillFilters, filters }) {
     vertical
   } = filters
   return (
-    <CardKPI title={'Filters'} actions={false} overflow={'auto'} textAlign={'center'}>
-        <Grid container>
-            <Grid item xs={12} sm={10} lg={6}>
+    <CardKPI title={'Filters'} actions={false} overflow={'auto'} textAlign={'start'}>
+        <Grid container >
+            <Grid item xs={12} sm={10} md ={6} lg={4} xl={4} style={{marginTop: 20}}>
                 <Sectors setFilters={setFilters} fillFilters={fillFilters} selectedList={sector}/>
             </Grid>
-            <Grid item xs={12} sm={10} lg={6}>
+            <Grid item xs={12} sm={10} md ={6} lg={4} xl={4} style={{marginTop: 20}}>
                 <Investor setFilters={setFilters} fillFilters={fillFilters} selectedList={investor}/>
             </Grid>
-            <Grid item xs={12} sm={10} lg={6}>
+            <Grid item xs={12} sm={10} md ={6} lg={4} xl={4} style={{marginTop: 20}}>
                <Size setFilters={setFilters}fillFilters={fillFilters} selectedList={size}/>
             </Grid>
-            <Grid item xs={12} sm={10} lg={6}>
+            <Grid item xs={12} sm={10} md ={6} lg={6} xl={6} style={{marginTop: 50}}>
                 <Growth setFilters={setFilters} fillFilters={fillFilters} selectedList={growth}/>
             </Grid>
-            <Grid item xs={12} sm={10} lg={12}>
+            <Grid item xs={12} sm={10} md ={6} lg={12} xl={10} style={{marginTop: 20}}>
```

Comment:
Can you reduce the space, so we can have in the same row all posible filters?

Reply:
solved comment